packages/system-api/src/show-notification.ts

```
@@ -0,0 +1,31 @@
+
+/*
+ * Copyright (c) 2011-2018 Genestack Limited
+ * All Rights Reserved
+ * This source code is distributed under the MIT license:
+ * https://github.com/genestack/JS-SDK/tree/master/LICENSE
+ */
+
+import {systemCall} from './system-calls';
+import {
+    NotificationType,
+    NotificationMessage,
+    SHOW_NOTIFICATION_MESSAGE_TYPE
+} from '@genestack/interfaces';
+
+export function showNotification(
+    message: string,
+    type: NotificationType = NotificationType.warning
+): Promise<any> {
+    const systemCallMessage: NotificationMessage = {
+        type: SHOW_NOTIFICATION_MESSAGE_TYPE,
+        payload: {
+            message,
+            type
+        }
+    };
+
+    systemCall(systemCallMessage);
+
+    return Promise.resolve()
```

Do we not interested in result of `systemCall`? Looks like in this place we suppress possible errors and asynchrony of `showNotification` is redundant. Why promise is used here?

Also `Promise.resolve()` returns `Promise<void>`. You should remove `: Promise<any>` from `showNotification` declaration (type inference would work) or replace it with `: Promise<void>`. Using of `any` should be avoided in the most cases.

---

My opinion: we do not need this. I am resolving `undefined` here because of that.